input/fsh/GenomicStudy.fsh

```diff
@@ -42,6 +42,14 @@ Description: "A Genomic Study is a set of files and procedures that are performe
   * protocolDefinition 0..* Reference(ActivityDefinition) "The defined protocol that describes the analysis"
   //* protocolPerformed 0..* CodeableReference -- note, not part of R4
   * protocolPerformed 0..* Reference(Procedure) "The protocol that was actually performed for the analysis"
+  * performer 0..* Reference(Practitioner 
+                             or PractitionerRole 
+                             or Organization 
+                             or CareTeam 
```

Comment:
I suppose this is OK, but I think we have to have some very clearly documented use cases to cast this wide of a net for 'performer' - I would suggest we stay in alignment with DiagnosticReport, Observation, ImagingStudy (or is that where this list came from?)

Reply:
I agree. We should base the allowed types on use cases. I'm particularly interested in how we will support non-clinical reporting use cases. Those performing genomic studies will not be limited to clinical staff.

Reply:
So, checked Observation:
[Reference](http://build.fhir.org/references.html#Reference)([Practitioner](http://build.fhir.org/practitioner.html) | [PractitionerRole](http://build.fhir.org/practitionerrole.html) | [Organization](http://build.fhir.org/organization.html) | [CareTeam](http://build.fhir.org/careteam.html) | [Patient](http://build.fhir.org/patient.html) | [RelatedPerson](http://build.fhir.org/relatedperson.html))

DiagnosticReport:
[Reference](http://build.fhir.org/references.html#Reference)([Practitioner](http://build.fhir.org/practitioner.html) | [PractitionerRole](http://build.fhir.org/practitionerrole.html) | [Organization](http://build.fhir.org/organization.html) | [CareTeam](http://build.fhir.org/careteam.html))

ImagingStudy:
'performer.actor'
[Reference](http://build.fhir.org/references.html#Reference)([Practitioner](http://build.fhir.org/practitioner.html) | [PractitionerRole](http://build.fhir.org/practitionerrole.html) | [Organization](http://build.fhir.org/organization.html) | [CareTeam](http://build.fhir.org/careteam.html) | [Patient](http://build.fhir.org/patient.html) | [Device](http://build.fhir.org/device.html) | [RelatedPerson](http://build.fhir.org/relatedperson.html) | [HealthcareService](http://build.fhir.org/healthcareservice.html))

Reply:
So in retrospect, we probably are not as far off from others as I thought we might be, but still need guidance as to why we need these. Saying 'because other resources' is a good start :)

Reply:
Great. I believe having performer as part of the genomicAnalysis element while the interpreter is a direct element of the genomicStudy is comparable to the imaging study and may give room for non-clinical staff to be represented as part of the genoimcAnalysis.
Currently, we can represent used devices in two elements, 1) as part of the performer element, and 2) as one of the values of device element. This is comparable to imagingStudy modality and performer elements. In the genomics use cases, do rename device to sequencingDevice for example, and others to be represented as perfomer(s)?

Reply:
Well Observation keeps device as its own attribute and doesn't allow a device as performer, and ImagingStudy says a device can be a performer. I am not sure why the difference to be honest, but it might be a good question to ask about the distinction. It might be a legal/regulatory thing about the difference between the device that 'generated' data versus 'performing the test' ?

Reply:
According to R4, ImagingStudy.series.modality is a coding that uses "CID 29 Acquisition Modality" codes as valueset. While ImagingStudy.series.performer.actor could be Device resource, among others.
We may raise this point during Thursday meeting.
One more interesting element is the ImagingStudy.series.performer.function which is a codeableconcept. We may need to start thinking of creating a similar value set for various performers roles who may participate in the genomicAnalysis.